packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div
         :class="$style['c-rating']"
-        data-test-id="rating-component">
+        :data-test-id="`${setRatingVariant}-component`">
```

Comment:
Isn't this computed prop more of a `get` than a `set`?

Reply:
Fixed.